libs/clients/taktikal/README.md

```diff
@@ -0,0 +1,38 @@
+# Taktikal
+
+## About
+
+The client allows its users to request individuals to sign PDF documents using their mobile electronic ids via Taktikal.
+
+## Usage
+
+Add `TaktikalModule` to your Module imports:
+
+```typescript
+import { TaktikalModule, TaktikalService } from '@island.is/clients/taktikal'
+
+@Module({
+  imports: [
+    TaktikalModule.register({
+      basePath: 'https://onboardingdev.taktikal.is/api',
+      companyKey: '<company key>',
+      apiKey: '<api key>',
+    }),
+  ],
+  providers: [
+    TaktikalService
+  ]
```

Comment:
You don't need to add the service in providers, as the `TaktikalModule` exports it.